packages/lib/bookings/filterHostsByLeadThreshold.ts

```diff
@@ -49,6 +49,10 @@ const computeLeadOffsets = async <T = Record<string, unknown>>({
     ) => {
       // satisfy TS, obviously as we've where'd on userId it cannot be null but try telling TS that..
       if (!booking.userId) return acc;
+      if (!hosts.map((host) => host.user.id).includes(booking.userId)) {
```

Comment:
The following clause allows for bookingCounts to be included that are not the current host, it's when the host is part of another Booking where he/she is not the organizer. This is an extra failsafe this never happens

```typescript
    {
      attendees: {
        some: {
          email: {
            in: users.map((user) => user.email),
          },
        },
      },
    },
```